src/hotspot/os/windows/os_windows.cpp

```diff
@@ -4090,6 +4090,39 @@ int    os::win32::_build_minor               = 0;
 bool   os::win32::_processor_group_warning_displayed = false;
 bool   os::win32::_job_object_processor_group_warning_displayed = false;
 
+void GetWindowsInstallationType(char* buffer, int bufferSize) {
```

Comment:
Nit: we don't use this naming style in hotspot. This should be either `getWindowsInstallationType` or `get_windows_installation_type`. Thanks

Reply:
updated